app/models/manageiq/providers/azure_stack/cloud_manager/orchestration_template.rb

```diff
@@ -0,0 +1,105 @@
+class ManageIQ::Providers::AzureStack::CloudManager::OrchestrationTemplate < ::OrchestrationTemplate
```

Comment:
This entire file is a copy-paste from Azure provider, not sure whether I should explicitly give credit in a comment or something?